src/helper.py

```diff
@@ -42,48 +42,55 @@ def arguments_dictionary(lat, long, city, args):
 
 
-def set_output_values(args, arguments):  # noqa
-    """
-    Takes a list of command line arguments(args)
-    and sets the appropritate values
-    in the arguments dictionary(show_wave = 1, etc).
-    Returns the arguments dict with the updated CLI args
-    """
-    if "hide_wave" in args or "hw" in args:
-        arguments["show_wave"] = 0
-    if "show_large_wave" in args or "slw" in args:
-        arguments["show_large_wave"] = 1
-    if "hide_uv" in args or "huv" in args:
-        arguments["show_uv"] = 0
-    if "hide_height" in args or "hh" in args:
-        arguments["show_height"] = 0
-    if "hide_direction" in args or "hdir" in args:
-        arguments["show_direction"] = 0
-    if "hide_period" in args or "hp" in args:
-        arguments["show_period"] = 0
-    if "hide_location" in args or "hl" in args:
-        arguments["show_city"] = 0
-    if "hide_date" in args or "hdate" in args:
-        arguments["show_date"] = 0
-    if "metric" in args or "m" in args:
-        arguments["unit"] = "metric"
-    if "json" in args or "j" in args:
-        arguments["json_output"] = 1
-    if "gpt" in args or "g" in args:
-        arguments["gpt"] = 1
-    if "show_air_temp" in args or "sat" in args:
-        arguments["show_air_temp"] = 1
-    if "show_wind_speed" in args or "sws" in args:
-        arguments["show_wind_speed"] = 1
-    if "show_wind_direction" in args or "swd" in args:
-        arguments["show_wind_direction"] = 1
+def set_output_values(args, arguments):
+    """
+    Takes a list of command line arguments (args)
+    and sets the appropriate values
+    in the arguments dictionary (show_wave = 1, etc).
+    Returns the arguments dict with the updated CLI args.
+    """
+    actions = {
+        "hide_wave": ("show_wave", 0),
+        "hw": ("show_wave", 0),
+        "show_large_wave": ("show_large_wave", 1),
+        "slw": ("show_large_wave", 1),
+        "hide_uv": ("show_uv", 0),
+        "huv": ("show_uv", 0),
+        "hide_height": ("show_height", 0),
+        "hh": ("show_height", 0),
+        "hide_direction": ("show_direction", 0),
+        "hdir": ("show_direction", 0),
+        "hide_period": ("show_period", 0),
+        "hp": ("show_period", 0),
+        "hide_location": ("show_city", 0),
+        "hl": ("show_city", 0),
+        "hide_date": ("show_date", 0),
+        "hdate": ("show_date", 0),
+        "metric": ("unit", "metric"),
+        "m": ("unit", "metric"),
+        "json": ("json_output", 1),
+        "j": ("json_output", 1),
+        "gpt": ("gpt", 1),
+        "g": ("gpt", 1),
+        "show_air_temp": ("show_air_temp", 1),
+        "sat": ("show_air_temp", 1),
+        "show_wind_speed": ("show_wind_speed", 1),
+        "sws": ("show_wind_speed", 1),
+        "show_wind_direction": ("show_wind_direction", 1),
+        "swd": ("show_wind_direction", 1),
+    }
+
+    for arg in args:
+        if arg in actions:
+            key, value = actions[arg]
+            arguments[key] = value
```

Comment:
Great work!
IMO, there's one point I'd like to highlight. To ensure that the modifications you've made haven't caused any regressions (and that everything is functioning correctly), I recommend adding test cases to test_helper.py. This will help guarantee the integrity of the changes.

Reply:
Hey @ryansurf Thank you!

I only made changes to the helper.py function and cli.py function where I fixed a typo. I am not sure how files like idea/misc.xml were altered. 
I will also try to figure out why the formatter is failing but if not, I will let you know

Reply:
> Hey @ryansurf Thank you!
> 
> I only made changes to the helper.py function and cli.py function where I fixed a typo. I am not sure how files like idea/misc.xml were altered. I will also try to figure out why the formatter is failing but if not, I will let you know

When you do `git add <file>`, make sure you're only adding `helper.py` and `cli.py`!

Reply:
Hey @GUKWAT, hows it going? Feel free to ping me if you need any help!

Reply:
Hey @ryansurf I am just working on the unit tests for helper.py. Still trying to figure out how to format the code using ruff having issues when it comes to the file path

Reply:
@GUKWAT great! What are the file path errors you are dealing with?